lib/postgres.js

```diff
@@ -4,5 +4,6 @@ var settings = require('./settings');
 
 exports.pg = knex({
   client: 'pg',
+  pool: { min: 4, max: 10 },
```

Comment:
Is there any chance we'd need to change these on the fly? Should we pull from env variables but use 4 and 10 as the fallback defaults?